tests/numpy/test_insert.py

```diff
@@ -0,0 +1,92 @@
+from unittest import skipIf
+
+try:
+    import pandas as pd
+
+    PANDAS_IMPORTED = True
+except ImportError:
+    PANDAS_IMPORTED = False
+
+#from tests.testcase import BaseTestCase
+from tests.numpy.testcase import NumpyBaseTestCase
+
+
+@skipIf(not PANDAS_IMPORTED, reason="pandas cannot be imported")
```

Comment:
Please see how simple numpy/pandas tests are implemented: tests/numpy/test_generic.py. Skipping logic (`@skipIf`) is hidden under the hood. Variable `PANDAS_IMPORTED` is redundant in this case.